### PR TITLE
Add repartition of tweets per user data and graph

### PR DESCRIPTION
**Pull request by martinratinaud**
Following the read of https://medium.com/dfrlab/twitter-accounts-disproportionately-amplified-chilean-presidential-candidate-hashtags-6f513369be8d

Let's add some (already existing) data to the user tab
Screen_Shot_2022-01-26_at_11_19_35
Screen Shot 2022-01-26 at 11 31 50

### CODE REVIEW

**Comment by martinratinaud** [comment]
@clementbiron can you review and propose something as I realize the bar chart does not display when max tweets is big

Something like
- 1 tweet
- 2 - 5 tweets
- 6 - 20 tweet
- + 200

I don't really like this approach as those threshholds introduce a bias

maybe something more like 
- all digits until 200
- then 200+ 

So that we keep max number of bars to 201

need advice, thanks

**Comment by clementbiron** [comment]
UserserTable headers should be on several lines to ensure the visibility of the text (as you did it for the % of all the tweets column).

image

File: src/modules/Common/components/Datatables/UsernameTable.tsx [hunk 1]
```diff
@@ -85,12 +85,32 @@ const UsernameTable = ({
...
+      ),
       accessor: 'value',
       align: 'right',
       size: 1,
       Cell: ({ value }: any) => value.toLocaleString('en'),
     },
+    {
+      Header: (
+        <>
+          % of all
```
Comment (clementbiron) [inline src/modules/Common/components/Datatables/UsernameTable.tsx:103]: https://github.com/ambanum/information-manipulation-analyzer/pull/62#issuecomment-1021951269

File: src/modules/Common/data-components/Username.tsx [hunk 2]
```diff
@@ -74,11 +87,26 @@ const DataUsername = ({
...
             y: value,
           }))}
+          xAxis={{
+            categories: [...Array(botRepartition.length)].map((_, index) => `${index}%`),
+          }}
+        />
+      </div>
+      <div className="fr-mt-8w">
+        <BarGraph
+          title="Nb of tweets per user"
```
Comment (clementbiron) [inline src/modules/Common/data-components/Username.tsx:104]: ~~Nb of tweets per user~~ -> `Tweets per user`
~~In number of users, views on bar chart~~-> `Views on bar chart`

File: src/modules/Common/data-components/Username.tsx [hunk 3]
```diff
@@ -74,11 +87,26 @@ const DataUsername = ({
...
+          subtitle="In number of users, views on bar chart"
```
Comment (clementbiron) [inline src/modules/Common/data-components/Username.tsx:105]: We need 4 strings configuration for this component :
- title ✅
- subtitle ✅
- y axis legend : ❌ 
- x axis legend : ❌

Comment (clementbiron) [reply src/modules/Common/data-components/Username.tsx:105]: In this case, y axis legend string could be `Nb of users`and x axis legend `Nb of Tweets`

**Comment by clementbiron** [comment]
> @clementbiron can you review and propose something as I realize the bar chart does not display when max tweets is big
> 
> Something like
> 
>     * 1 tweet
> 
>     * 2 - 5 tweets
> 
>     * 6 - 20 tweet
> 
>     * * 200
> 
> 
> I don't really like this approach as those threshholds introduce a bias
> 
> maybe something more like
> 
>     * all digits until 200
> 
>     * then 200+
> 
> 
> So that we keep max number of bars to 201
> 
> need advice, thanks

As we have seen together, let's cut into 6 groups :
- 1 tweets
- 2-5 tweets
- 6-10 tweets
- 11-50 tweets
- 51-200 tweets
- 200 tweets +

File: src/modules/Common/data-components/Username.tsx (hunk 3 above)
Comment (martinratinaud) [reply src/modules/Common/data-components/Username.tsx:105]: No need for xaxis legend as it uses the title of graph for now

File: src/modules/Common/components/Datatables/UsernameTable.tsx (hunk 1 above)
Comment (martinratinaud) [reply src/modules/Common/components/Datatables/UsernameTable.tsx:103]: I have an error right now so I merge without this change

**Pull request merged**